.gitignore

```diff
@@ -4,4 +4,4 @@ build/
 *.log
 node_modules/
 package-lock.json
-*.swp
+*.swp
```

Comment:
This looks like an unwanted edit.

Reply:
I've added a newline to the end of the file to remove this change